Analytics/gunicornserver.py

```diff
@@ -1,39 +1,62 @@
+from typing import Union
+
 from flask_script import Command, Option
 
+from settings.get_config_decorator import GetConfig
+
+
+@GetConfig('GunicornServer', 'gunicorn_server')
 class GunicornServer(Command):
+    """
+    Startup Green Unicorn server
+    """
 
     description = 'Run the app within Gunicorn'
 
-    def __init__(self, host='<host-value>', port=5000, workers=4):
-        self.port = port
-        self.host = host
-        self.workers = workers
+    def __init__(self, host: Union[None, str] = None,
+                 port: Union[None, int] = None,
+                 workers: Union[None, int] = None):
+        """
+        Set Host, Port and Worker count for Gunicorn server
+        :param host: Host URI
+        :param port: Network port
+        :param workers: Number of workers
+        """
+        if host:
+            self.gunicorn_host = host
```

Comment:
if any of these are null, then the self.gunicorn_<var> will not exist and error when you try to call it. Either remove the if or set a value in an else